nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -460,7 +460,8 @@ def add_electrical_series(
     use_times: bool = False,
     write_as: str = 'raw',
     es_key: str = None,
-    write_scaled: bool = False
+    write_scaled: bool = False,
+    use_compression: bool = True
```

Comment:
```suggestion
    compression: Optional[str] = "gzip"
```
@bendichter Sure, how about this, with `None` meaning no compression?